randomsentencebot.py

```diff
@@ -67,7 +67,7 @@ def get_twitter():
     global TWITTER
 
     if TWITTER is None:
-        data = load_yaml(args.yaml)
+        data = load_yaml(randomsentencebot_example.yaml)
```

Comment:
Let's not change this line: it's a variable, not a file name (which would be a string). We don't need to edit the `.py` file for this PR.

```suggestion
        data = load_yaml(args.yaml)
```